packages/fast-form-generator-react/src/form/form-item.children.tsx

```diff
@@ -234,15 +234,15 @@ class FormItemChildren extends React.Component<IFormItemChildrenProps & IManaged
         // we have nothing to add or delete
         if (items.length === 0) {
             return [
-                (<li key={0}>No actions available</li>)
+                (<li key={0}><span>No actions available</span></li>)
```

Comment:
if this is for styling can we target a class instance instead of the element?

Reply:
Seems like unneeded weight/complexity to add classes for everything, but I can change...

Reply:
was able to use another preexisting class